app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt

```diff
@@ -1,16 +1,337 @@
 package org.feature.fox.coffee_counter.ui.profile
 
-import androidx.compose.foundation.layout.Column
-import androidx.compose.runtime.Composable
+import android.widget.Toast
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.*
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
 import org.feature.fox.coffee_counter.R
+import org.feature.fox.coffee_counter.data.local.database.tables.User
 import org.feature.fox.coffee_counter.ui.common.MoneyAppBar
+import org.feature.fox.coffee_counter.ui.theme.CrayolaBrown
 
 @Preview(showSystemUi = true)
 @Composable
 fun ProfileView() {
-    Column {
+    val user = User(id = "a", name = "Max Mustermann", false, "123456789")
+    Column(
+        modifier = Modifier.padding(5.dp),
+        verticalArrangement = Arrangement.spacedBy(16.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
         MoneyAppBar(title = stringResource(R.string.profile_title))
+        ProfileIcon()
+        UserIdRow(user.id)
+        UserNameRow(user.name)
+        PasswordRow(user.password)
+        RetypePasswordRow(user.password)
```

Comment:
The Textfields with the Labels could be refactored in general.
Use the same Textfields as we use inside the AuthenticationView to stay consistent.
If you do that you could get rid of the labels that you used and also reduce LOC.

Reply:
Since this is not a LogIn Screen and therefore the user data should be displayed, a hint inside the text field can not be used as label because the text fields are not empty at beginning

Reply:
The solution inside the AuthenticationView automatically provides a Label for the Textfield if there is a text inside.

Reply:
Should be resolved with commit 0358f206-37ad-4acb-b08d-4155135c9d98

Reply:
Should be resolved with 68043254c9bf398ad297900a56d12ea722f934ee